### DISCUSSION TITLE
Training custom dataset on pretrained wheights

### DISCUSSION THREAD

**Discussion by eliastemponi** [ultralytics/yolov5#4206] [Q&A]
Hi.

I have a beginner doubt. I appreciate if anyone could help.

If I train a custom datasets from a pretrained (yolov5s.pt) model, Will I be able to detect all origin objects of coco model ? Or will it be able to detect only my custom data ? 

thks

**Comment by glenn-jocher** [comment]
@eliastemponi your model is trained to detect the classes of the dataset you provide.
